frontend/src/pages/ShowProject.tsx

```diff
@@ -0,0 +1,28 @@
+import ProjectCard from "../components/ProjectCard.tsx";
+import {useFetch} from "../hooks/useFetch.tsx";
+import {useParams} from "react-router-dom";
+
+
+export default function ShowProject() {
+
+    const {id} = useParams();
+    const getProjectById = useFetch(state => state.getProjectById);
+
+
+    if (!id) {
+        throw new Error("Id is undefined")
+    }
+
+    const project = getProjectById(id);
+
+    if (!project) {
+        throw new Error(`No project with id ${id} found`)
```

Comment:
hier wird die selbe Error Message wie bereits in frontend/src/hooks/useFetch.tsx angezeigt. Ich weiß nicht ob das gewollt ist.

Reply:
Habe mich selbst schon daran gestört 🙃
Tatsächlich bekomme ich sonst aber eine Meldung:
"TS2322: Type 'Project | undefined' is not assignable to type 'Project'.   Type 'undefined' is not assignable to type 'Project'."
in der ShowProject.tsx 🤷‍♂️
Muss ich mir nochmal genauer anschauen.